aatrostle/buttchat
# Feature/express socketio chat

### This PR contains the following:

#### Features
- Add `express` and custom server
- Add `socker.io` and configure with server
- Add basic in memory persistence
- Add basic chat experience with server hydration, client fetching and websocket updates
- Style updates

#### Bug fixes
- Fix issue with `/about` rendering `/` content

#### How to verify/test
Prior to pulling this PR the above features didn't exist. The features aren't formally specified either so to test, just use it as you might expect. We will apply more deliberate design to this feature later on.

Feel free to report issues or suggestions.

#### Dependencies
None
